src/containers/Navbar2/Navbar.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback } from 'react'
+
+import { NetworkAlert } from 'decentraland-ui/dist/components/NetworkAlert/NetworkAlert'
+import { Navbar2 as NavbarComponent } from 'decentraland-ui/dist/components/Navbar2/Navbar2'
+import { getChainName } from '@dcl/schemas/dist/dapps/chain-id'
+import { Network } from '@dcl/schemas/dist/dapps/network'
+import { getAnalytics } from '../../modules/analytics/utils'
+import { t } from '../../modules/translation/utils'
+import { getBaseUrl } from '../../lib/utils'
+import ChainProvider from '../ChainProvider'
+import {
+  DROPDOWN_MENU_BALANCE_CLICK_EVENT,
+  DROPDOWN_MENU_DISPLAY_EVENT,
+  DROPDOWN_MENU_SIGN_OUT_EVENT
+} from '../UserInformation/constants'
+import { NavbarProps } from './Navbar.types'
+import { NAVBAR_CLICK_EVENT } from './constants'
+import UnsupportedNetworkModal from '../UnsupportedNetworkModal'
+
+const BASE_URL = getBaseUrl()
+
+const Navbar: React.FC<NavbarProps> = ({
+  appChainId,
+  docsUrl = 'https://docs.decentraland.org',
+  enablePartialSupportAlert = true,
+  isSwitchingNetwork,
+  ...props
+}: NavbarProps) => {
+  const expectedChainName = getChainName(appChainId)
+  const analytics = getAnalytics()
+
+  const handleSwitchNetwork = useCallback(() => {
+    props.onSwitchNetwork(appChainId)
+  }, [])
+
+  const handleClickBalance = useCallback(
+    (e: React.MouseEvent, network: Network) => {
+      e.preventDefault()
+      analytics.track(DROPDOWN_MENU_BALANCE_CLICK_EVENT, { network })
+
+      setTimeout(() => {
+        window.open(`${BASE_URL}/account`, '_blank', 'noopener')
+      }, 300)
+    },
+    [analytics]
+  )
+
+  const handleClickNavbarItem = useCallback(
+    (
+      _e: React.MouseEvent,
```

Comment:
If any of the navbar item clicks imply a redirection or going to another page, if we don't wait for the tracking to be finished, it won't be tracked.

Reply:
here is a timeOut defined when this `CB` is called: https://github.com/decentraland/ui/blob/master/src/components/Navbar2/SubMenuItem/SubMenuItem.tsx#L20-L33
```
const handleClick = useCallback(
    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
      event.preventDefault()
      onClickMenuOption &&
        onClickMenuOption(event, { eventTrackingName, url: href, isExternal })
      setTimeout(
        () => {
          window.open(href, isExternal ? '_blank' : '_self', 'noopener')
        },
        onClickMenuOption ? 300 : 0
      )
    },
    [onClickMenuOption, eventTrackingName, href, isExternal]
  )
```